layers/ubuntu1604/java/BUILD

```diff
@@ -118,9 +117,9 @@ genrule(
     srcs = ["tests.yaml.tpl"],
     outs = ["tests.yaml"],
     cmd = (
-        # JDK version 10 includes a '+' sign.
-        # TODO: remove for future versions of Java that do not need it
-        "sed 's|{_JAVA_REVISION_ENCODED}|" + OPENJDK_INSTALLER.revision.replace("+", "%2B") + "|g; s|{_JAVA_REVISION}|" + OPENJDK_INSTALLER.revision + "|g' <$< >$@"
+        "sed 's|{_SRC_REVISION}|" + OPENJDK_SRC.revision.rpartition("/")[2] +
```

Comment:
Would appreciate a comment here explaining what the sed command is doing with a small example if possible

Reply:
Added comments. Let me know if it's not good.